articles/creating-your-first-google-web-story/index.md

```diff
@@ -0,0 +1,276 @@
+Introduced in 2018 as [AMP](https://amp.dev/) (Accelerated Mobile Pages) story, Google Web Story is owned by Google and powered by AMP technology. AMP (Accelerated Mobile Pages), developed by the AMP Open-Source team, is an open-source HTML platform designed for mobile web browsers to load web pages faster.
+
+### Table of Contents
+
+[Creating your first Google Web Story](#creating-your-first-google-web-story)
+[Table of Contents](#table-of-contents)
+[What is a Web Story?](#what-is-a-web-story)
+[Prerequisites](#prerequisites)
+[What differentiates a Web Story?](#what-differentiates-a-web-story)
+[Creating a Web Story](#creating-a-web-story)
+[Using NoCode](#using-nocode)
+[Using Code](#using-code)
+[How to make your Web Story visible on the Web](#how-to-make-your-web-story-visible-on-the-web)
+[Conclusion](#conclusion)
+[Further Reading](#further-reading)
+
+### What is a Web Story?
+
+> This is a web-based version of the popular “Stories” format that uses images, audio, videos, texts, and animations to create an engaging short consumable experience.
+
+The web prefix signifies the content is opened to everyone on the web and it is also short-lived or evergreen, depending on how the owner wants it to last.
+
+### Prerequisites
+
+To get started with Web Stories you’ll need some basic Knowledge of HTML, CSS, and JavaScript or no prior knowledge, a code editor to write the code, and a web browser to preview the code.
+
+### What differentiates a Web Story?
+
+Compared the conventional social app stories, Web stories are similar in terms of the overall consistent user experience of tapping back and forth to view previous and next stories. But some things that differentiate Web stories from social app stories include;
+
+**Control**
+Contrary to social app stories with Web stories, you're in control of how long the story should be on the Web and not having to worry that your beautifully crafted story would expire after 24 hours. The publisher host Web stories, which means the content, the copyright, and rules depend on the publisher's terms of choice.
+
+**Earn money**
+In almost all social app stories, creators don't earn when a story has an ad, but this is different for Web stories. Creators can earn money for ads on the story pages, as well as brand deals, native advertising, and so on.
+
+**Accessibility**
+Since Web stories live on the web, they don't just live in a single app but can be accessed anywhere. Technically Web stories are Web pages under the hood, which makes them get indexed by search engines.
+
+### Creating a Web Story
+
+Creating a Web Story is flexible depending on your needs and expertise. We can create Web Stories using Code to create a story with custom functionalities and NoCode using drag-and-drop tools.
+
+#### Using NoCode
```

Comment:
Use level three ```###``` headings rather than ```####```

Reply:
This is a subheading of a h3 tag that's why I used h4. what do you recommend I use?